cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/AddCommand.java

```diff
@@ -130,15 +130,15 @@ public void execute() {
             CommandUtil.printError(errStream,
                     "The following required arguments were not provided:\n" +
                             "    <module-name>",
-                    "ballerina add <module-name> [-t|--template <template-name>]",
+                    "ballerina add <module-name> [-t|--template <template-name>].",
```

Comment:
Isn't this an example of the command itself? I don't think we need the period at the end.

Reply:
+1. Sorry, we don't need the period here.

Reply:
Fixed.